db/migrate/20201210083801_add_column_to_friendships.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToFriendships < ActiveRecord::Migration[5.2]
+  def change
+    add_column :friendships, :requester_id
```

Comment:
I got an error while running this migration 👇 .

![image](https://user-images.githubusercontent.com/57726348/102378121-75bb3680-3feb-11eb-9ea5-df00aee446ab.png)

You forgot to mention the data type. Please rectify this.